### PR TITLE
polish(pu): delete unused enable_fast_timestep argument

### PR DESCRIPTION
**Pull request by puyuan1996**
## Description
- delete unused enable_fast_timestep argument

## Related Issue
https://github.com/opendilab/DI-engine/issues/851
## Check List

- [x] merge the latest version source branch/repo, and resolve all the conflicts
- [x] pass style check
- [x] pass all the tests

### CODE REVIEW

File: ding/model/template/q_learning.py [hunk 1]
```diff
@@ -943,31 +952,35 @@ def __init__(
...
-                transitions, then we would use this list to indicate how to save and return hidden state.
-        ArgumentsKeys:
-            - obs (:obj:`torch.Tensor`): The raw observation tensor.
-            - prev_state (:obj:`list`): The previous rnn state tensor, whose structure depends on ``lstm_type``.
+            - inputs (:obj:`Dict`): Input data dictionary containing observation and previous RNN state.
+            - inference (:obj:`bool`): If True, unrolls one timestep (used during evaluation). If False, unrolls \
+              the entire sequence (used during training).
+            - saved_state_timesteps (:obj:`Optional[list]`): When inference is False, specifies the timesteps \
+              whose hidden states are saved and returned.
+        Input Keys:
```
Comment (PaParaZz1) [inline ding/model/template/q_learning.py:964]: don't modify original keyword name

File: ding/model/template/q_learning.py [hunk 2]
```diff
@@ -943,31 +952,35 @@ def __init__(
...
+            - obs (:obj:`torch.Tensor`): Raw observation tensor.
+            - prev_state (:obj:`list`): Previous RNN state tensor, structure depends on ``lstm_type``.
+
         Returns:
-            - outputs (:obj:`Dict`): The output of DRQN's forward, including logit (q_value) and next state.
-        ReturnsKeys:
-            - logit (:obj:`torch.Tensor`): Discrete Q-value output of each possible action dimension.
-            - next_state (:obj:`list`): The next rnn state tensor, whose structure depends on ``lstm_type``.
+            - outputs (:obj:`Dict`): Dictionary containing the Q-value logits and next RNN state.
```
Comment (PaParaZz1) [inline ding/model/template/q_learning.py:969]: Q-value logits is not proper, maybe the original content is more reasonable

File: ding/model/template/q_learning.py [hunk 3]
```diff
@@ -943,31 +952,35 @@ def __init__(
...
-            - inference: (:obj:'bool'): Whether to enable inference forward mode, if True, we unroll the one timestep \
-                transition, otherwise, we unroll the eentire sequence transitions.
-            - saved_state_timesteps: (:obj:'Optional[list]'): When inference is False, we unroll the sequence \
-                transitions, then we would use this list to indicate how to save and return hidden state.
-        ArgumentsKeys:
-            - obs (:obj:`torch.Tensor`): The raw observation tensor.
-            - prev_state (:obj:`list`): The previous rnn state tensor, whose structure depends on ``lstm_type``.
+            - inputs (:obj:`Dict`): Input data dictionary containing observation and previous RNN state.
+            - inference (:obj:`bool`): If True, unrolls one timestep (used during evaluation). If False, unrolls \
+              the entire sequence (used during training).
```
Comment (PaParaZz1) [inline ding/model/template/q_learning.py:961]: there needs an indent

File: ding/model/template/q_learning.py [hunk 4]
```diff
@@ -855,18 +855,23 @@ def reshape(d):
...
+        The DRQN (Deep Recurrent Q-Network) is a neural network model combining DQN with RNN to handle sequential
+        data and partially observable environments. It consists of three main components: ``encoder``, ``rnn``,
+        and ``head``.
+        - **Encoder**: Extracts features from various observation inputs.
+        - **RNN**: Processes sequential observations and other data.
+        - **Head**: Computes Q-values for each action dimension.
+
     Interfaces:
-        ``__init__``, ``forward``.
+        - ``__init__``: Initializes the DRQN model.
```
Comment (PaParaZz1) [inline ding/model/template/q_learning.py:866]: recover to the original format here

File: ding/model/template/q_learning.py [hunk 5]
```diff
@@ -884,43 +889,47 @@ def __init__(
     ) -> None:
         """
         Overview:
-            Initialize the DRQN Model according to the corresponding input arguments.
+            Initializes the DRQN model with specified parameters.
```
Comment (PaParaZz1) [inline ding/model/template/q_learning.py:892]: This is an imperative sentence; we shouldn't use the third person singular form.

**Review by PaParaZz1 — APPROVED** [review]
